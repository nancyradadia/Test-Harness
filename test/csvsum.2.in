1, 2, 3, -90
67, 89, 98, -100