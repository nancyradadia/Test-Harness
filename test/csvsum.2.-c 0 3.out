Column 0 sum: 68.0
Column 3 sum: -190.0